lgc/patch/Patch.cpp

```diff
@@ -246,7 +254,7 @@ void Patch::addOptimizationPasses(legacy::PassManager &passMgr) {
     passMgr.add(createIndVarSimplifyPass());
     passMgr.add(createLoopIdiomPass());
     passMgr.add(createLoopDeletionPass());
-    passMgr.add(createSimpleLoopUnrollPass(optLevel));
+    passMgr.add(createSimpleLoopUnrollPass(cl::OptLevel));
```

Comment:
Notice that cl::OptLevel is equal to CodeGenOpt::Default (i.e. 2) by default. So this patch would be changing the current optimization level used (i.e. 3).

Reply:
I ran some experiments and this change of default optimization level (from 3 to 2) didn't translate into any differences in the output code so I think this is fine.